src/LuaORM/Model.lua

```diff
@@ -36,13 +36,22 @@ Model.targetTable = nil
 --
 -- @tparam mixed[] _tableConfiguration The table configuration of this Model's table
 --
+-- @tparam table _modelOptions The options for this model. (optional)
+--
 -- @treturn Model The Model instance
 --
-function Model:__construct(_tableConfiguration)
+function Model:__construct(_tableConfiguration, _modelOptions)
 
   local instance = setmetatable({}, {__index = Model})
   instance.targetTable = Table(_tableConfiguration)
-  instance:createTable()
+
+  if _modelOptions == nil then
+    _modelOptions = {}
+  end
+
+  if _modelOptions.createTable == nil or _modelOptions.createTable ~= false then
+    instance:createTable()
+  end
```

Comment:
Please add a else case here that calls `instance.targetTable:validate()`.
That method does not execute any queries, it only checks if the config for the table is valid and auto adjusts some config values if required, also it logs some warning messages that help the users to find problems in their configurations.

Reply:
Done and tested on my environment.